custom_components/powercalc/data/signify/LWE002/model.json

```
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "TP-Link Tapo P110 (via Home Assistant)",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 1,
+        "SLEEP_TIME": 25,
+        "VERSION": "v0.19.7:docker"
+    },
+    "name": "Hue White E14 SES Candle 470lm w/ BT (LWE002)",
+    "standby_power": 0.0,
```

```suggestion
    "standby_power": 0.25,
```

Standby power can not be 0.
0.25 would be a good approximation.